blocks/header/header.js

```diff
@@ -1,9 +1,33 @@
 import { getMetadata, decorateIcons } from '../../scripts/lib-franklin.js';
-import { createTagList } from '../../scripts/scripts.js';
 
 // media query match that indicates mobile/tablet width
 const isDesktop = window.matchMedia('(min-width: 900px)');
 
+const placeholderHtml = `
+<div class="header block" data-block-name="header" data-block-status="loading">
+  <dialog class="nav-wrapper">
+    <nav id="nav" >
+      <div class="nav-hamburger">
+        <button type="button" aria-controls="nav" aria-label="Open navigation">
+          <span class="nav-hamburger-icon"></span>
+        </button>
+      </div>
+      <div class="nav-brand" style="width: 80px; height: 99px;">
+        <p>
+        </p>
+      </div>
+      <div class="nav-sections" style="width: 424px; height: 38px;">
+        <ul>
+          <li>test</li>
```

Comment:
Do you really need to put content as the <ul> ? Is this required to minimize the layout shift ?

Reply:
removed the `<li>` tags, and added the style to the `<ul>`. No layout shift and no content